src/features/consent/Consent.tsx

```diff
@@ -53,40 +65,46 @@ export default function Consent() {
           children={''}
         />
       </Box>
+
       <Box id="summary">
-        <ul>
-          <li>
-            <h6>Owned by {protectedDataSelected?.owner}</h6>
-          </li>
-          <li>
-            <h6>Data Protected Address: {protectedDataSelected?.address}</h6>
-          </li>
+        <ul
```

Comment:
Minor change to UI: Make items more readable = slightly bigger font size than before.